lib/Backends/CPU/LLVMIRGen.cpp

```diff
@@ -1318,7 +1319,10 @@ void LLVMIRGen::generateLLVMIRForInstr(llvm::IRBuilder<> &builder,
     unsigned unrollDFactor = 1;
 
     if ((destDepth % 8) == 0) {
-      unrollDFactor = 8;
+      bool groupDividedBy8 = (destDepth / 8) % CI->getGroup() == 0;
```

Comment:
Please add parenthesis here:
((destDepth / 8) % CI->getGroup()) == 0;

The reader may read this code late at night, and remembering the order of operators is one more mental task (what comes first, % or == ?)

Reply:
Also, a comment could be nice.